.env.example

```diff
@@ -8,4 +8,7 @@ STORAGE_CONTAINER_NAME
 STRAPI_DATABASE_CONNECTION_NAME
 STRAPI_DATABASE_NAME
 STRAPI_DATABASE_USERNAME
-STRAPI_DATABASE_PASSWORD
\ No newline at end of file
+STRAPI_DATABASE_PASSWORD
+SENDGRID_API_KEY
+SENDGRID_DEFAULT_FROM_ADDRESS
+SENDGRID_DEFAULT_REPLY_TO_ADDRESS
```

Comment:
I would like to see a mention of sendgrid in the readme and where these values can be acquired from.